packages/material-renderers/src/complex/MaterialTableControl.tsx

```diff
@@ -57,9 +57,11 @@ import {
   encode,
   ArrayTranslations,
 } from '@jsonforms/core';
-import DeleteIcon from '@mui/icons-material/Delete';
-import ArrowDownward from '@mui/icons-material/ArrowDownward';
-import ArrowUpward from '@mui/icons-material/ArrowUpward';
+import {
+  Delete as DeleteIcon,
+  ArrowDownward,
+  ArrowUpward,
+} from '@mui/icons-material';
```

Comment:
Are these icon changes necessary?

Reply:
For me this was necessary. With updating the typescript version, the old imports no longer worked. This was the case during the build as well in the IDE. Curiously that already happens when going from TS 4.2.3 to 4.2.4. I looked into the changelog and there as a change about resolving something. So that could be the difference.